src/function.py

```diff
@@ -336,8 +336,9 @@ def is_private_func_start(self, block: tac_cfg.TACBasicBlock) -> t.Dict[
                     return None
                 if len(pre.delta_stack) == 0:
                     return None
-                for val in list(pre.delta_stack):
-                    ref_block = self.cfg.get_block_by_ident(str(val))
```

Comment:
This seems a bit iffy. An approach that uses `cfg.get_blocks_by_pc`  instead of `cfg.get_block_by_ident` seems like it might be more robust.

If `val.is_const` is false then `val.identifier` will return something that cannot be a block identifier in any case, so I'm not sure I see the purpose of this ternary expression. Seems like it will be questionable when duplicated blocks exist as well.

Reply:
If val.is_const is false, then we want the `V*` identifier. The ternary expression is because val.const_value is `None` if the value is not a constant, which means you can't use it in `hex()`.

I will need to look into a better way of handling duplicated blocks. We probably need to change the CFG class.